sdk/python/tests/nem/test_TransactionFactory.py

```diff
@@ -23,27 +21,31 @@ class TransactionFactoryTest(unittest.TestCase):
 
 	def test_can_create_known_transaction_from_descriptor(self):
 		# Arrange:
-		for transaction_name, transaction_type in [('transfer', 0x0101), ('importance-transfer', 0x0801)]:
-			factory = TransactionFactory(FOO_NETWORK)
+		for transaction_name, transaction_type in [('transfer_transaction_v1', 0x0101), ('account_key_link', 0x0801)]:
+			factory = TransactionFactory(Network.TESTNET)
 
 			# Act:
-			transaction = factory.create({'type': transaction_name})
+			transaction = factory.create({
+				'type': transaction_name,
+				'signer_public_key': bytes(32)
+			})
 
 			# Assert:
-			self.assertEqual(transaction_type, transaction.type)
-			self.assertEqual(0x54000001, transaction.version)
+			self.assertEqual(transaction_type, transaction.type_.value)
+			self.assertEqual(0x98, transaction.network.value)
+			self.assertEqual(1, transaction.version)
```

Comment:
use same order (network, version) or (version, network) everywhere. reverse order is used in BatchOperations.

Reply:
(version, network)